React/Views/RCTWrapperViewController.m

```diff
@@ -117,7 +117,7 @@ - (void)viewWillAppear:(BOOL)animated
     bar.translucent = _navItem.translucent;
     bar.titleTextAttributes = _navItem.titleTextColor ? @{
       NSForegroundColorAttributeName: _navItem.titleTextColor
-    } : nil;
+    } : bar.titleTextAttributes;
```

Comment:
This should be rewritten in manner where we are modify only NSForegroundColorAttributeName value inside the dictionary.